### PR TITLE
add getPerfectNumbers spec&impl

### PR DESCRIPTION
**Pull request by codeshakti**
Please check it;

### CODE REVIEW

**Comment by marcrg** [comment]
It's Ok!!!

File: stuff/javascript/jasmine-standalone-2.8.0/spec/getPerfectNumbersSpec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+describe('getPerfectNumbers', function() {
+    it('should return a 0', function() {
```
Comment (manuelbarzi) [inline stuff/javascript/jasmine-standalone-2.8.0/spec/getPerfectNumbersSpec.js:2]: TODO: change the it('should... to more semantic one, like the following:

it('should return a "is a perfect number" for a perfect number input') {

File: stuff/javascript/jasmine-standalone-2.8.0/spec/getPerfectNumbersSpec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+        var n = 6;
+        
+        var res = getPerfectNumbers(n);
+
+        expect(res).toBe(n + ' is a perfect number');
+    });
+
+    it('should return an error if is not a perfect number', function() {
```
Comment (manuelbarzi) [inline stuff/javascript/jasmine-standalone-2.8.0/spec/getPerfectNumbersSpec.js]: WARN! is not true that the function should return an error. instead it should return a message "<n> is not a perfect number". it must be corrected!

TODO: change the it('should... by the following:

it('should return "is not a perfect number" for a non-perfect number input') {

**Comment by mangione77** [comment]
Ok!

**Review by manuelbarzi — CHANGES_REQUESTED** [review]
TODO: apply corrections, and then it will be merged. @codeshakti please, take a look!

**Review by manuelbarzi — APPROVED** [review]

**Pull request merged**